### PR TITLE
ReleaseNotes: Updated changelog and release notes for 8.0.0-beta3

### PR DESCRIPTION
**Pull request by grafanabot**
This exciting! So much has changed!

### CODE REVIEW

**Review by hugohaggmark — APPROVED** [review]
LGTM!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,68 @@
 
+<!-- 8.0.0-beta3 START -->
+
+# 8.0.0-beta3 (2021-06-01)
+
+### Features and enhancements
+
+* ** CloudWatch:** Add Lambda@Edge Amazon CloudFront metrics. [#34561](https://github.com/grafana/grafana/pull/34561), [@razor-x](https://github.com/razor-x)
+* ** Time series panel:** Position tooltip correctly when window scrolled or resized. [#34782](https://github.com/grafana/grafana/pull/34782), [@dprokop](https://github.com/dprokop)
```
Comment (dprokop) [inline CHANGELOG.md:9]: Nit: These two need a PR rename after this is merged I think

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,68 @@
...
+* **Azure:** Expose azure settings to external plugins. [#34484](https://github.com/grafana/grafana/pull/34484), [@sunker](https://github.com/sunker)
+* **AzureMonitor:** Deprecate using separate credentials for Logs. [#34758](https://github.com/grafana/grafana/pull/34758), [@andresmgot](https://github.com/andresmgot)
+* **AzureMonitor:** Display variables in Logs resource picker. [#34648](https://github.com/grafana/grafana/pull/34648), [@joshhunt](https://github.com/joshhunt)
+* **AzureMonitor:** Hide App Insights for data sources not using it. [#34725](https://github.com/grafana/grafana/pull/34725), [@joshhunt](https://github.com/joshhunt)
+* **AzureMonitor:** Support querying Subscriptions and Resource Groups in Logs. [#34766](https://github.com/grafana/grafana/pull/34766), [@joshhunt](https://github.com/joshhunt)
+* **AzureMonitor:** remove requirement for default subscription. [#34787](https://github.com/grafana/grafana/pull/34787), [@kostrse](https://github.com/kostrse)
+* Clarify that Threema Gateway Alerts support only Basic IDs. [#34828](https://github.com/grafana/grafana/pull/34828), [@dbrgn](https://github.com/dbrgn)
```
Comment (dprokop) [inline CHANGELOG.md:18]: Rename needed - no scope defined

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,68 @@
...
+
+
+![image](https://user-images.githubusercontent.com/46142/119644192-8383ac00-be14-11eb-9b17-6eb0217f958c.png)
+![image](https://user-images.githubusercontent.com/46142/119644260-96967c00-be14-11eb-8489-a82ef2c2c443.png)
+
+
+**Which issue(s) this PR fixes**:
+
+Fixes #33709
+ Issue [#34725](https://github.com/grafana/grafana/issues/34725)
```
Comment (dprokop) [inline CHANGELOG.md:62]: hmm, this looks wrong?

File: CHANGELOG.md (hunk 1 above)
Comment (dprokop) [reply CHANGELOG.md:9]: @hugohaggmark updated both of these

File: CHANGELOG.md (hunk 2 above)
Comment (dprokop) [reply CHANGELOG.md:18]: updated

File: CHANGELOG.md (hunk 1 above)
Comment (hugohaggmark) [reply CHANGELOG.md:9]: Thank you!

File: CHANGELOG.md (hunk 2 above)
Comment (hugohaggmark) [reply CHANGELOG.md:18]: Thank you!

File: CHANGELOG.md (hunk 3 above)
Comment (hugohaggmark) [reply CHANGELOG.md:62]: Yeah

Comment (hugohaggmark) [reply CHANGELOG.md:62]: @joshhunt can you check why https://github.com/grafana/grafana/pull/34725 is ending up in the deprecations part of the release notes. Maybe try not to use the word deprecation? I'm not sure.

Comment (joshhunt) [reply CHANGELOG.md:62]: 🤦🏻 The inline text "Deprecation notices" was enough to trigger this getting added here.

I've updated the PR description - should this re-run and it'll regenerate?

Comment (hugohaggmark) [reply CHANGELOG.md:62]: I'm just guessing, but I think that the next time we run the update change log action your changes to the description will show @joshhunt. Thank you for helping out!

**Pull request closed without merging**